lib/globals.dart

```diff
@@ -124,4 +124,5 @@ List<List<Color>> colors = [
   [Colors.green[700]             , Colors.green                  ],
 ];
 
+const String APP_DIR                                    = 'C:/Users/Alex Mercer/Music';
```

Comment:
You probably have to remove this here and change it to default user.
But I don't know anything about dart ;-)

Reply:
Yeah 😅. There isn't a automatic way to do it on Windows yet. I'll try to write some more C++ or just ask user to set the directory.

Reply:
Yes! There will be .exe files for Windows. The plan is to add support for Linux aswell.